pages/apis/rest_api/annotations.md

```diff
@@ -107,6 +109,13 @@ Optional [request body properties](/docs/api#request-body-properties):
       <p class="Docs__api-param-eg"><em>Example:</em> <code>"info"</code></p>
     </td>
   </tr>
+  <tr>
+    <th><code>priority</code></th>
+    <td>
+      Priority of the annotation (1 to 10). By default annotations have a priority of 3. Annotations with a priority of 10 will be shown first, and annotations with a priority of 1 will be shown last.
```

Comment:
Thanks for your contribution @thejcannon ! 🎉 

Just some minor suggested wording tweaks before merging :-) and I'll approve this PR in the meantime.

```suggestion
      The priority of the annotation (`1` to `10`). Annotations with a priority of `10` are shown first, while annotations with a priority of `1` are shown last. When this option is not specified, annotations have a default priority of `3`.
```

Reply:
Oh yeah totally. I just ripped the wording straight from the Buildkite agent CLI docs.

Reply:
Ah, thanks for pointing this out!
I've created https://github.com/buildkite/agent/pull/2670 to get this text updated too (since it's generated in the docs from the Buildkite Agent source code).